ci/docker/linux-apt-docs.dockerfile

```diff
@@ -55,7 +57,10 @@ RUN apt-get update -y && \
         r-base=${r}* \
         rsync \
         ruby-dev \
-        wget && \
+        wget \
+        libasound2-dev && \
+    wget -q https://freeshell.de/phd/chromium/jammy/pool/latest/chromium_124.0.6367.60~linuxmint1+virginia_amd64.deb && \
+    apt-get install -y ./chromium_124.0.6367.60~linuxmint1+virginia_amd64.deb && \
```

Comment:
Can I try https://packages.debian.org/search?keywords=chromium ?
(Use Debian instead of Ubuntu as base image and use Debian's chromium.)

Reply:
Since we were already using ubuntu as the base image, I didn't want to change the docs base image like that. we can try shifting the whole thing to debian instead though, which is a pretty big change. The docs image uses `ubuntu-cpp` and `ubuntu-python` as bases, I'd have to change those to use debian instead of ubuntu also.

If you feel like its warranted, give it a try! :smile:

Reply:
Is there a particular reason to change everything up to use debian instead?

Reply:
If we can somehow use an official chromium package and (probably) not require the `libasound2-dev` to build documentation, but maybe we can do that on a different PR. It is great to add the mermaid integration but those dependencies seem brittle

Reply:
yea. unfortunately even the mermaid-cli requires puppeteer which launches a headless chrome instance which requires libasound2-dev for whatever reason. 

The "official" chromium package ends up failing to launch, saying it needs to be installed via snap. So this is the upstream chromium package from linuxmint (an ubuntu based distro) and was the common solution I found for people who wanted to install chromium without using snap.

There's effort for mermaid-cli to be able to run without chromium as a dependency, but it seems like it's going to be a while before that happens.

Reply:
https://freeshell.de/phd/chromium/jammy/pool/latest/chromium_124.0.6367.60~linuxmint1+virginia_amd64.deb will be invalid eventually. Our CI will be failed after that.

Reply:
I created an issue for `sphinxcontrib-mermaid`: https://github.com/apache/arrow/issues/41430

Reply:
@kou so are we punting changing to debian for a follow-up PR?

Reply:
Could you wait for #41455 and rebase on main after it's merged?

Reply:
Sure thing